packages/FormElement/src/FormElement.js

```diff
@@ -131,6 +131,14 @@ function FormElement(props) {
     "aria-invalid": hasError,
   };
 
+  const supportedComponents = ["DatePicker", "Input"];
```

Comment:
Currently we only support these "Paprika" components so we can be explicit about this. As we are using the `formElement` component as the root component for each question in results questionnaires (which gets passed many different types of children) it would be best to ensure the extendedProps only get added to the correct child type.